distribution/docs/src/main/resources/_contents/_endpoints/metrics-endpoint-contents.adoc

```diff
@@ -0,0 +1,221 @@
+====
+
+The Metrics Endpoint is used by the <<Metrics Collection Application>> to report on system metrics.
+
+===== Installing the Metrics Endpoint
+
+The Metrics Endpoint is installed by default with a standard installation in the ${ddf-platform} application.
+
+===== Configuring the Metrics Endpoint
+
+No configuration can be made for the Metrics Endpoint.
+All of the metrics that it collects data on are pre-configured in ${branding}.
+
+===== Using the Metrics Endpoint
+
+.Metrics Endpoint URL
+[source,http]
+----
+${secure_url}/services/internal/metrics/catalogQueries.png?startDate=2013-03-31T06:00:00-07:00&endDate=2013-04-01T11:00:00-07:00
+----
+
+The table below lists all of the options for the Metrics endpoint URL to execute custom metrics data requests:
+
+.Metrics Endpoint URL Options
+[cols="1m,5,3" options="header"]
+|===
+
+|Parameter
+|Description
+|Example
+
+|startDate
+|Specifies the start of the time range of the search on the metric's data (RFC-3339 - Date and Time format, i.e. YYYY-MM-DDTHH:mm:ssZ). Date/time must be earlier than the endDate. +
+_This parameter cannot be used with the `dateOffset` parameter._
+|`startDate=2013-03-31T06:00:00-07:00`
+
+|endDate
+|Specifies the endof the time range of the search on the metric's data (RFC-3339 - Date and Time format, i.e. `YYYY-MM-DDTHH:mm:ssZ`). Date/time must be later than the startDate. +
+_This parameter cannot be used with the `dateOffset` parameter._
+|`endDate=2013-04-01T11:00:00-07:00`
+
+|dateOffset
+|Specifies an offset, backwards from the current time, to search on the modified time field for entries. Defined in seconds and must be a positive Integer. +
+_This parameter cannot be used with the `startDate` or `endDate` parameters._
+|`dateOffset=1800`
+
+|yAxisLabel
+|(optional) the label to apply to the graph's y-axis. Will default to the metric's name, e.g., Catalog Queries. +
+_This parameter is only applicable for the metric's graph display format.
+|Catalog Query Count
+
+|title
+|(optional) the title to be applied to the graph.
+
+Will default to the metric's name plus the time range used for the graph.
+
+_This parameter is only applicable for the metric's graph display format._
+|Catalog Query Count for the last 15 minutes
+
+|===
+
+====== Metrics Data Supported Formats
+
+The metric's historical data can be displayed in several formats, including PNG , a CSV file, an Excel .xls file, a PowerPoint .ppt file, an XML file, and a JSON file.
+The PNG, CSV, and XLS formats are accessed via hyperlinks provided in the Metrics tab web page.
+The PPT, XML, and JSON formats are accessed by specifying the format in the custom URL, e.g., \http://<${branding}_HOST>:<${branding}_PORT>/services/internal/metrics/catalogQueries.json?dateOffset=1800.
+
+The table below describes each of the supported formats, how to access them, and an example where applicable. (NOTE: all example URLs begin with \${secure_url} which is omitted in the table for brevity.)
+
+.Metrics Formats
```

Comment:
This table looks terrible in PDF format. We may want to consider putting the examples outside the table to make things readable, or have separate sub-sections for each format and not use a table at all.